tests/frameworks/gtest/include/gtest/gtest.h

```diff
@@ -478,6 +480,23 @@ class GTEST_API_ Test {
   GTEST_DISALLOW_COPY_AND_ASSIGN_(Test);
 };
 
+// Helper TestRunner class used to indirect from a TAEF test class which is named with a Prefix to a single
+// class that TEST_P can friend. TODO: move to a impl file to not violate ODR (doesn't matter since all defs are the same though)
+class TestRunner {
```

Comment:
>TestRunner [](start = 6, length = 10)

Uh I don't remember why I felt I needed this. I should try to remove it. I think the metafactory change was sufficient.